app/models/transcript.rb

```diff
@@ -23,6 +23,27 @@ class Transcript < ActiveRecord::Base
   has_many :transcript_edits
   has_many :transcript_speakers
 
+  def speakers
+    return "" if transcript_speakers.blank?
+    transcript_speakers.includes(:speaker).pluck(:name).join("; ") + "; "
+  end
+
+  def speakers=(params)
+    return unless valid?
```

Comment:
If the transcript update or create triggers a validation exception and the form is rendered again then the speakers would have been updated, so I added this guard clause is to only execute the code when the transcript is valid.